pkg/api/rest/update.go

```diff
@@ -44,6 +48,22 @@ type RESTUpdateStrategy interface {
 	AllowUnconditionalUpdate() bool
 }
 
+// TODO: add other common fields that require global validation.
+func commonFieldsValidation(obj, old runtime.Object) error {
+	objectMeta, err := api.ObjectMetaFor(obj)
+	if err != nil {
+		return errors.NewInternalError(err)
+	}
+	oldObjectMeta, err := api.ObjectMetaFor(old)
+	if err != nil {
+		return errors.NewInternalError(err)
+	}
```

Comment:
Maybe just call the validate object meta update function here instead of reimplementing.